lib/hope-app-link.js

```diff
@@ -0,0 +1,124 @@
+/*global fxosComponent */
```

Comment:
we should probably move this component into the `homescreen` dir.
long term we might have a completely separate prototyping track looking at the homescreen/new tab page, maybe a separate repo etc...